ntbs-ui-tests/Features/ImportLegacyNotification.feature

```diff
@@ -22,4 +22,20 @@ Feature: Import legacy notification
     And I can see the value 963 657 7830 for element with id 'banner-nhs-number'
     And I can see the value Male for element with id 'banner-sex'
     And I can see the value Sensitive to first line for element with id 'banner-drug-resistance'
-    And I can see the value Complete for element with id 'banner-treatment-outcome'
\ No newline at end of file
+    And I can see the value Complete for element with id 'banner-treatment-outcome'
+    
+  @CookieOverride
+  Scenario: Import legacy notification - Cookie Override
+    Given I navigate to the app
+    And I have logged in as BirminghamServiceUser
+    And I am on the Search page
+    When I enter 100894 into 'SearchParameters_IdFilter'
+    And I click on the 'search-button' button
+    And I wait
+    And I click on the 100894 link
+    And I click on the 'import-button' button
+    And I wait
+
+    Then A new notification should have been created
```

Comment:
We could add a few more checks in here (notification date and tb-service perhaps?) but I'm erring on the side of caution here and not committing any user data at all